backend/battle/tests/test_view.py

```diff
@@ -0,0 +1,123 @@
+from django.test import TestCase, Client
+from model_bakery import baker
+from django.urls import reverse
+from battle.models import Team
+from users.models import User
+
+
+class AssociatePokemonsToBattleTest(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.client = Client()
+        self.user = User.objects.create(email='example@example.com')
+        self.user.set_password('admin')
+        self.user.save()
+        self.battle = baker.make('battle.Battle', creator=self.user)
+        names = ['pikachu', 'bulbasaur', 'pidgeot', 'wrong_name']
+        for pokemon_name in names:
+            baker.make("pokemon.Pokemon", name=pokemon_name, attack=60, defense=15, hp=10)
+
+    def test_user_can_create_team_with_pokemons(self):
+        self.client.login(username=self.user.email, password='admin')
```

Comment:
I suggested on some other PR to extract this to an utils class

Reply:
I updated it in this PR, does it have some problem?